superset-frontend/package.json

```diff
@@ -81,7 +81,7 @@
     "@superset-ui/legacy-plugin-chart-treemap": "^0.11.15",
     "@superset-ui/legacy-plugin-chart-word-cloud": "^0.11.15",
     "@superset-ui/legacy-plugin-chart-world-map": "^0.11.15",
-    "@superset-ui/legacy-preset-chart-big-number": "^0.11.21",
+    "@superset-ui/legacy-preset-chart-big-number": "^0.12.13",
```

Comment:
This seemed to need a bump... not seeing any problems with it, but flagging it just in case.

Reply:
Yeah, `^0.12.13` is probably identical to `^0.11.21`. It was republished since moved into the `superset-ui` repo but the actual code doesn't really change (except the storybook).

Reply:
Thanks, @ktmud :)

If you don't mind reviewing/approving the [superset-ui PR](https://github.com/apache-superset/superset-ui/pull/352), I'll merge both, unless you see any deployment problems in doing so.